when this gets called the stack is likely not there; avoid the tlb flushes


--- glibc-2.28/nptl/allocatestack.c~	2018-08-01 05:10:47.000000000 +0000
+++ glibc-2.28/nptl/allocatestack.c	2018-09-27 16:18:09.043648892 +0000
@@ -385,6 +385,8 @@
 {
   uintptr_t sp = (uintptr_t) CURRENT_STACK_FRAME;
   size_t pagesize_m1 = __getpagesize () - 1;
+
+  return;
 #if _STACK_GROWS_DOWN && !defined(NEED_SEPARATE_REGISTER_STACK)
   size_t freesize = (sp - (uintptr_t) mem) & ~pagesize_m1;
   assert (freesize < size);
